arangod/RocksDBEngine/RocksDBLogValue.cpp

```diff
@@ -108,43 +108,39 @@ RocksDBLogValue RocksDBLogValue::BeginTransaction(TRI_voc_tick_t dbid,
   return RocksDBLogValue(RocksDBLogType::BeginTransaction, dbid, tid);
 }
 
-RocksDBLogValue RocksDBLogValue::DocumentOpsPrologue(TRI_voc_cid_t cid) {
-  return RocksDBLogValue(RocksDBLogType::DocumentOperationsPrologue, cid);
+RocksDBLogValue RocksDBLogValue::CommitTransaction(TRI_voc_tick_t dbid,
+                                                   TRI_voc_tid_t tid) {
+  return RocksDBLogValue(RocksDBLogType::CommitTransaction, dbid, tid);
 }
 
-RocksDBLogValue RocksDBLogValue::DocumentRemove(
-    arangodb::StringRef const& key) {
-  return RocksDBLogValue(RocksDBLogType::DocumentRemove, key);
-}
-
-RocksDBLogValue RocksDBLogValue::DocumentRemoveAsPartOfUpdate(
-    arangodb::StringRef const& key) {
-  return RocksDBLogValue(RocksDBLogType::DocumentRemoveAsPartOfUpdate, key);
+RocksDBLogValue RocksDBLogValue::DocumentRemoveV2(TRI_voc_rid_t rid) {
```

Comment:
I thought about adding more info to that marker, but then again: 1.) It can only ever appear after a BeginTransaction marker  2) It is ignored otherwise
So I though adding more information to it would be redundant and only make transactions slower

Reply:
Okay, so say we need to replay a delete, and do not have an `objectId` mapping for the collection in question. The `BeginTransaction` marker tells us which database we are in, but not which collection. Is this a problem?

I don't think so, as the main reason we would not have an `objectId` mapping is if the collection itself was later dropped. This should only cause an issue for recovery with external stores like iResearch, but I think we solved the issue there by ignoring those deletes and handling it at the collection drop level.

So it's probably okay?

Reply:
yes, if the collection isn't there (`shouldHandleCollection` returns `false`) the marker is ignored.